config/initializers/ext_action_mailer.rb

```diff
@@ -11,5 +13,65 @@ def deliver_now_or_later(opts = {})
       end
       # rubocop:enable IdentityIdp/MailLaterLinter
     end
+
+    def deliver_later(opts = {})
+      MailerSensitiveInformationChecker.check_for_sensitive_pii!(@params, @args, @action)
+      original_deliver_later(opts)
+    end
+  end
+end
+
+class MailerSensitiveInformationChecker
```

Comment:
I would move this to its own file (in lib?) class since it's no longer an extension method on ActionMailer

Reply:
good call, moved in [8280a26](https://github.com/18F/identity-idp/pull/7155/commits/8280a26a21f9d559a646a5f180ef944af3d72bd6)